packages/mobile/ios/AudiusReactNative.xcodeproj/xcshareddata/xcschemes/Staging.xcscheme

```diff
@@ -59,15 +59,6 @@
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
-      <PreActions>
```

Comment:
oh interesting , what's context?

Reply:
This was an empty script I noticed in the scheme, just cleaning it up!